### ISSUE TITLE
Downstream training loss and accuracy are nearly constants

### ISSUE THREAD

**Issue by bnbsking** [MCG-NJU/VideoMAE#8]
Hello, thanks for your effort on coding VideoMAE. I have followed the installation guide completely, then I selected two categories from UCF-101 to do downstream training (video binary classification).  The data is preprocessed including resized the short edge to 240px or 320px as well as saved in .mp4 format. The train.csv, val.csv, test.csv are all prepared also. However, how could the training loss /accuracy are nearly contstants 0.69/0.5 throughout the training? 

The dataset I specified is UCF-101 and the nb_classes is set as 2. The only other thing I modfied from the official script is the learning rate due to working on single GPU non-distributed environment. I have tried different multiple setting of learning rate from 64x to 0.015x, but all of them leads to constant losses. Could you please help me, thank you a lot.

![Screenshot from 2022-05-09 11-06-49](https://user-images.githubusercontent.com/43401779/167335553-080068a8-dad8-4cfb-876b-2ccc66b0fee0.png)

**Comment by yztongzhan** [comment]
Hi @bnbsking!  Thanks for your question. I don't know if our VideoMAE can work well when the number of training videos is not enough. You can try to tarin a linear classifier on top of the frozen encoder.

**Comment by bnbsking** [comment]
Hello @yztongzhan. Would you mind to upload the pretext training weight of UCF-101?

I plot to use the entire UCF-101 dataset to do downstream training, and hope to solve the constant loss issue. Thanks a lot.
